[2025-03-13T14:23:02.632+0000] {processor.py:186} INFO - Started process (PID=37) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:23:02.633+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:23:02.637+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:02.636+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:23:02.646+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:23:02.680+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.057 seconds
[2025-03-13T14:23:22.864+0000] {processor.py:186} INFO - Started process (PID=38) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:23:22.866+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:23:22.869+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:22.869+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:23:22.874+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:23:22.901+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.044 seconds
[2025-03-13T14:23:53.213+0000] {processor.py:186} INFO - Started process (PID=39) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:23:53.216+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:23:53.219+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:53.219+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:23:53.224+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:23:53.242+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.041 seconds
[2025-03-13T14:23:54.258+0000] {processor.py:186} INFO - Started process (PID=40) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:23:54.259+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:23:54.264+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:54.263+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:23:54.283+0000] {processor.py:925} INFO - DAG(s) 'etl_reddit_pipeline' retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:23:54.440+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:54.440+0000] {override.py:1930} INFO - Created Permission View: can read on DAG:etl_reddit_pipeline
[2025-03-13T14:23:54.451+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:54.449+0000] {override.py:1930} INFO - Created Permission View: can delete on DAG:etl_reddit_pipeline
[2025-03-13T14:23:54.459+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:54.458+0000] {override.py:1930} INFO - Created Permission View: can edit on DAG:etl_reddit_pipeline
[2025-03-13T14:23:54.467+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:54.466+0000] {override.py:1930} INFO - Created Permission View: can create on DAG Run:etl_reddit_pipeline
[2025-03-13T14:23:54.474+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:54.474+0000] {override.py:1930} INFO - Created Permission View: menu access on DAG Run:etl_reddit_pipeline
[2025-03-13T14:23:54.480+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:54.480+0000] {override.py:1930} INFO - Created Permission View: can read on DAG Run:etl_reddit_pipeline
[2025-03-13T14:23:54.489+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:54.488+0000] {override.py:1930} INFO - Created Permission View: can delete on DAG Run:etl_reddit_pipeline
[2025-03-13T14:23:54.489+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:54.489+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-13T14:23:54.503+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:54.503+0000] {dag.py:3262} INFO - Creating ORM DAG for etl_reddit_pipeline
[2025-03-13T14:23:54.520+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:23:54.519+0000] {dag.py:4180} INFO - Setting next_dagrun for etl_reddit_pipeline to None, run_after=None
[2025-03-13T14:23:54.550+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.299 seconds
[2025-03-13T14:24:24.866+0000] {processor.py:186} INFO - Started process (PID=41) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:24:24.870+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:24:24.874+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:24:24.873+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:24:24.890+0000] {processor.py:925} INFO - DAG(s) 'etl_reddit_pipeline' retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:24:24.921+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:24:24.921+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-13T14:24:24.954+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:24:24.954+0000] {dag.py:4180} INFO - Setting next_dagrun for etl_reddit_pipeline to None, run_after=None
[2025-03-13T14:24:24.987+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.135 seconds
[2025-03-13T14:24:44.027+0000] {processor.py:186} INFO - Started process (PID=42) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:24:44.028+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:24:44.031+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:24:44.031+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:24:44.041+0000] {processor.py:925} INFO - DAG(s) 'etl_reddit_pipeline' retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:24:44.160+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:24:44.160+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-13T14:24:44.187+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:24:44.187+0000] {dag.py:4180} INFO - Setting next_dagrun for etl_reddit_pipeline to 2025-03-12 00:00:00+00:00, run_after=2025-03-13 00:00:00+00:00
[2025-03-13T14:24:44.221+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.201 seconds
[2025-03-13T14:25:14.459+0000] {processor.py:186} INFO - Started process (PID=43) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:25:14.460+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:25:14.463+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:25:14.462+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:25:14.481+0000] {processor.py:925} INFO - DAG(s) 'etl_reddit_pipeline' retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:25:14.511+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:25:14.510+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-13T14:25:14.548+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:25:14.547+0000] {dag.py:4180} INFO - Setting next_dagrun for etl_reddit_pipeline to 2025-03-12 00:00:00+00:00, run_after=2025-03-13 00:00:00+00:00
[2025-03-13T14:25:14.578+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.129 seconds
[2025-03-13T14:25:31.651+0000] {processor.py:186} INFO - Started process (PID=44) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:25:31.653+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:25:31.655+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:25:31.655+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:25:31.665+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:25:31.663+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:25:31.666+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:25:31.690+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.046 seconds
[2025-03-13T14:26:01.957+0000] {processor.py:186} INFO - Started process (PID=45) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:26:01.958+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:26:01.961+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:26:01.960+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:26:01.974+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:26:01.973+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:26:01.975+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:26:02.001+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.055 seconds
[2025-03-13T14:26:04.003+0000] {processor.py:186} INFO - Started process (PID=46) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:26:04.015+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:26:04.028+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:26:04.027+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:26:04.049+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:26:04.046+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:26:04.050+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:26:04.105+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.124 seconds
[2025-03-13T14:26:34.267+0000] {processor.py:186} INFO - Started process (PID=47) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:26:34.269+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:26:34.271+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:26:34.271+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:26:34.276+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:26:34.275+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:26:34.277+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:26:34.291+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T14:27:04.509+0000] {processor.py:186} INFO - Started process (PID=48) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:27:04.510+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:27:04.513+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:27:04.513+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:27:04.521+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:27:04.520+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:27:04.521+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:27:04.542+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.038 seconds
[2025-03-13T14:27:34.802+0000] {processor.py:186} INFO - Started process (PID=49) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:27:34.803+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:27:34.805+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:27:34.804+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:27:34.811+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:27:34.811+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:27:34.812+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:27:34.833+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.037 seconds
[2025-03-13T14:28:05.034+0000] {processor.py:186} INFO - Started process (PID=50) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:28:05.035+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:28:05.040+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:28:05.039+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:28:05.046+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:28:05.045+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:28:05.047+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:28:05.068+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.042 seconds
[2025-03-13T14:28:35.259+0000] {processor.py:186} INFO - Started process (PID=51) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:28:35.261+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:28:35.264+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:28:35.263+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:28:35.271+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:28:35.270+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:28:35.271+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:28:35.296+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.047 seconds
[2025-03-13T14:29:05.492+0000] {processor.py:186} INFO - Started process (PID=52) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:29:05.495+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:29:05.498+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:29:05.497+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:29:05.506+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:29:05.504+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:29:05.508+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:29:05.540+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.055 seconds
[2025-03-13T14:29:35.723+0000] {processor.py:186} INFO - Started process (PID=53) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:29:35.725+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:29:35.728+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:29:35.727+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:29:35.742+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:29:35.740+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:29:35.742+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:29:35.773+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.064 seconds
[2025-03-13T14:30:05.986+0000] {processor.py:186} INFO - Started process (PID=54) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:30:05.987+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:30:05.989+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:30:05.988+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:30:05.994+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:30:05.993+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:30:05.994+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:30:06.011+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.034 seconds
[2025-03-13T14:30:36.235+0000] {processor.py:186} INFO - Started process (PID=55) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:30:36.236+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:30:36.238+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:30:36.237+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:30:36.243+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:30:36.243+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:30:36.243+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:30:36.259+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T14:31:06.507+0000] {processor.py:186} INFO - Started process (PID=56) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:31:06.508+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:31:06.510+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:31:06.509+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:31:06.515+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:31:06.514+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:31:06.515+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:31:06.531+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T14:31:36.775+0000] {processor.py:186} INFO - Started process (PID=57) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:31:36.776+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:31:36.778+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:31:36.777+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:31:36.783+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:31:36.782+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:31:36.783+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:31:36.800+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T14:32:07.028+0000] {processor.py:186} INFO - Started process (PID=58) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:32:07.029+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:32:07.030+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:32:07.030+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:32:07.036+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:32:07.035+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:32:07.036+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:32:07.049+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T14:32:37.318+0000] {processor.py:186} INFO - Started process (PID=59) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:32:37.319+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:32:37.320+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:32:37.320+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:32:37.325+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:32:37.325+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:32:37.326+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:32:37.339+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.026 seconds
[2025-03-13T14:33:07.575+0000] {processor.py:186} INFO - Started process (PID=60) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:33:07.576+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:33:07.577+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:33:07.577+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:33:07.583+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:33:07.582+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:33:07.583+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:33:07.597+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T14:33:37.832+0000] {processor.py:186} INFO - Started process (PID=61) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:33:37.833+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:33:37.835+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:33:37.834+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:33:37.841+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:33:37.840+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:33:37.841+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:33:37.861+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.036 seconds
[2025-03-13T14:34:08.060+0000] {processor.py:186} INFO - Started process (PID=62) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:34:08.062+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:34:08.064+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:34:08.063+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:34:08.070+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:34:08.069+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:34:08.070+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:34:08.088+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.033 seconds
[2025-03-13T14:34:38.524+0000] {processor.py:186} INFO - Started process (PID=63) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:34:38.527+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:34:38.529+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:34:38.529+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:34:38.544+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:34:38.540+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:34:38.544+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:34:38.570+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.054 seconds
[2025-03-13T14:35:08.761+0000] {processor.py:186} INFO - Started process (PID=64) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:35:08.767+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:35:08.769+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:35:08.769+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:35:08.776+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:35:08.775+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:35:08.777+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:35:08.797+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.042 seconds
[2025-03-13T14:35:38.997+0000] {processor.py:186} INFO - Started process (PID=65) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:35:38.998+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:35:38.999+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:35:38.999+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:35:39.005+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:35:39.004+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:35:39.006+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:35:39.021+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T14:36:09.197+0000] {processor.py:186} INFO - Started process (PID=66) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:36:09.198+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:36:09.199+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:36:09.199+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:36:09.206+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:36:09.205+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:36:09.207+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:36:09.222+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T14:36:39.368+0000] {processor.py:186} INFO - Started process (PID=67) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:36:39.369+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:36:39.370+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:36:39.370+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:36:39.376+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:36:39.375+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:36:39.377+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:36:39.393+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T14:37:09.695+0000] {processor.py:186} INFO - Started process (PID=68) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:37:09.697+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:37:09.700+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:37:09.700+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:37:09.714+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:37:09.712+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:37:09.714+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:37:09.749+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.068 seconds
[2025-03-13T14:37:39.924+0000] {processor.py:186} INFO - Started process (PID=69) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:37:39.925+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:37:39.927+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:37:39.926+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:37:39.932+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:37:39.932+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:37:39.933+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:37:39.950+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.032 seconds
[2025-03-13T14:38:10.179+0000] {processor.py:186} INFO - Started process (PID=70) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:38:10.180+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:38:10.182+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:38:10.181+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:38:10.188+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:38:10.187+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:38:10.188+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:38:10.204+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T14:38:40.346+0000] {processor.py:186} INFO - Started process (PID=71) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:38:40.347+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:38:40.348+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:38:40.348+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:38:40.354+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:38:40.353+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:38:40.354+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:38:40.368+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.027 seconds
[2025-03-13T14:39:10.595+0000] {processor.py:186} INFO - Started process (PID=72) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:39:10.596+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:39:10.597+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:39:10.597+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:39:10.604+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:39:10.603+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:39:10.604+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:39:10.620+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T14:39:40.777+0000] {processor.py:186} INFO - Started process (PID=73) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:39:40.779+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:39:40.780+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:39:40.780+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:39:40.786+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:39:40.785+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:39:40.786+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:39:40.800+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T14:40:11.055+0000] {processor.py:186} INFO - Started process (PID=74) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:40:11.059+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:40:11.061+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:40:11.060+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:40:11.073+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:40:11.071+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:40:11.074+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:40:11.102+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.063 seconds
[2025-03-13T14:40:41.299+0000] {processor.py:186} INFO - Started process (PID=75) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:40:41.300+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:40:41.304+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:40:41.303+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:40:41.315+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:40:41.313+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:40:41.316+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:40:41.338+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.049 seconds
[2025-03-13T14:41:11.560+0000] {processor.py:186} INFO - Started process (PID=76) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:41:11.567+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:41:11.574+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:41:11.572+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:41:11.607+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:41:11.603+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:41:11.610+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:41:11.676+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.140 seconds
[2025-03-13T14:41:41.731+0000] {processor.py:186} INFO - Started process (PID=77) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:41:41.732+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:41:41.734+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:41:41.733+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:41:41.739+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:41:41.739+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:41:41.740+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:41:41.756+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T14:42:11.946+0000] {processor.py:186} INFO - Started process (PID=78) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:42:11.949+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:42:11.950+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:42:11.950+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:42:11.962+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:42:11.960+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:42:11.963+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:42:11.991+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.054 seconds
[2025-03-13T14:42:42.216+0000] {processor.py:186} INFO - Started process (PID=79) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:42:42.220+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:42:42.223+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:42:42.222+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:42:42.241+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:42:42.240+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:42:42.242+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:42:42.286+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.084 seconds
[2025-03-13T14:43:12.693+0000] {processor.py:186} INFO - Started process (PID=80) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:43:12.696+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:43:12.698+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:43:12.697+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:43:12.710+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:43:12.709+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:43:12.710+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:43:12.735+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.049 seconds
[2025-03-13T14:43:43.016+0000] {processor.py:186} INFO - Started process (PID=81) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:43:43.017+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:43:43.019+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:43:43.018+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:43:43.028+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:43:43.027+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:43:43.028+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:43:43.108+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.097 seconds
[2025-03-13T14:44:21.419+0000] {processor.py:186} INFO - Started process (PID=37) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:44:21.419+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:44:21.423+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:44:21.423+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:44:21.434+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:44:21.434+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:44:21.435+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:44:21.454+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.041 seconds
[2025-03-13T14:44:51.697+0000] {processor.py:186} INFO - Started process (PID=38) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:44:51.698+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:44:51.700+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:44:51.700+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:44:51.706+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:44:51.705+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:44:51.707+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:44:51.722+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T14:45:21.952+0000] {processor.py:186} INFO - Started process (PID=39) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:45:21.953+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:45:21.955+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:45:21.955+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:45:21.960+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:45:21.960+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:45:21.961+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:45:21.975+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T14:45:52.202+0000] {processor.py:186} INFO - Started process (PID=40) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:45:52.203+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:45:52.206+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:45:52.205+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:45:52.211+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:45:52.210+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:45:52.211+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:45:52.229+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.034 seconds
[2025-03-13T14:46:22.424+0000] {processor.py:186} INFO - Started process (PID=41) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:46:22.425+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:46:22.427+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:46:22.427+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:46:22.432+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:46:22.432+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:46:22.433+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:46:22.449+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T14:46:52.681+0000] {processor.py:186} INFO - Started process (PID=42) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:46:52.683+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:46:52.690+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:46:52.688+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:46:52.717+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:46:52.713+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:46:52.719+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:46:52.753+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.088 seconds
[2025-03-13T14:47:22.921+0000] {processor.py:186} INFO - Started process (PID=43) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:47:22.922+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:47:22.925+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:47:22.925+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:47:22.937+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:47:22.936+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:47:22.938+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:47:22.977+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.064 seconds
[2025-03-13T14:47:53.235+0000] {processor.py:186} INFO - Started process (PID=44) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:47:53.237+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:47:53.243+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:47:53.242+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:47:53.256+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:47:53.255+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:47:53.257+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:47:53.286+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.064 seconds
[2025-03-13T14:48:23.471+0000] {processor.py:186} INFO - Started process (PID=45) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:48:23.472+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:48:23.475+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:48:23.475+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:48:23.481+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:48:23.480+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:48:23.481+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:48:23.497+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.032 seconds
[2025-03-13T14:48:53.708+0000] {processor.py:186} INFO - Started process (PID=46) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:48:53.709+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:48:53.712+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:48:53.712+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:48:53.718+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:48:53.717+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:48:53.719+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:48:53.734+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.032 seconds
[2025-03-13T14:49:23.999+0000] {processor.py:186} INFO - Started process (PID=47) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:49:24.000+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:49:24.002+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:49:24.002+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:49:24.007+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:49:24.006+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:49:24.007+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:49:24.022+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T14:49:54.299+0000] {processor.py:186} INFO - Started process (PID=48) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:49:54.300+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:49:54.302+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:49:54.302+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:49:54.307+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:49:54.307+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:49:54.308+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:49:54.324+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T14:50:24.539+0000] {processor.py:186} INFO - Started process (PID=49) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:50:24.540+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:50:24.543+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:50:24.542+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:50:24.549+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:50:24.548+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:50:24.549+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:50:24.564+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.034 seconds
[2025-03-13T14:50:54.781+0000] {processor.py:186} INFO - Started process (PID=50) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:50:54.781+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:50:54.784+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:50:54.784+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:50:54.789+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:50:54.789+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:50:54.790+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:50:54.806+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T14:51:25.032+0000] {processor.py:186} INFO - Started process (PID=51) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:51:25.033+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:51:25.036+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:51:25.035+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:51:25.043+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:51:25.042+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:51:25.043+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:51:25.060+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.034 seconds
[2025-03-13T14:51:55.256+0000] {processor.py:186} INFO - Started process (PID=52) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:51:55.257+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:51:55.259+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:51:55.259+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:51:55.265+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:51:55.264+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:51:55.266+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:51:55.281+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T14:52:25.547+0000] {processor.py:186} INFO - Started process (PID=53) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:52:25.548+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:52:25.550+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:52:25.550+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:52:25.556+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:52:25.555+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:52:25.556+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:52:25.572+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T14:52:55.798+0000] {processor.py:186} INFO - Started process (PID=54) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:52:55.799+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:52:55.801+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:52:55.801+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:52:55.806+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:52:55.806+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:52:55.807+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:52:55.822+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T14:53:26.095+0000] {processor.py:186} INFO - Started process (PID=55) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:53:26.095+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:53:26.097+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:53:26.097+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:53:26.103+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:53:26.102+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:53:26.103+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:53:26.119+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T14:53:56.405+0000] {processor.py:186} INFO - Started process (PID=56) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:53:56.406+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:53:56.409+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:53:56.408+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:53:56.414+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:53:56.413+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:53:56.414+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:53:56.429+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T14:54:26.716+0000] {processor.py:186} INFO - Started process (PID=57) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:54:26.717+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:54:26.719+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:54:26.719+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:54:26.724+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:54:26.724+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:54:26.725+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:54:26.740+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T14:54:57.052+0000] {processor.py:186} INFO - Started process (PID=58) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:54:57.053+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:54:57.056+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:54:57.055+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:54:57.061+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:54:57.060+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:54:57.061+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:54:57.075+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T14:55:27.275+0000] {processor.py:186} INFO - Started process (PID=59) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:55:27.276+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:55:27.278+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:55:27.278+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:55:27.284+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:55:27.283+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:55:27.284+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:55:27.298+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T14:55:57.508+0000] {processor.py:186} INFO - Started process (PID=60) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:55:57.509+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:55:57.511+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:55:57.511+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:55:57.516+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:55:57.516+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:55:57.517+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:55:57.532+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T14:56:27.797+0000] {processor.py:186} INFO - Started process (PID=61) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:56:27.798+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:56:27.800+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:56:27.800+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:56:27.806+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:56:27.805+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:56:27.806+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:56:27.820+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T14:56:58.081+0000] {processor.py:186} INFO - Started process (PID=62) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:56:58.081+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:56:58.083+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:56:58.083+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:56:58.088+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:56:58.087+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:56:58.088+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:56:58.102+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.027 seconds
[2025-03-13T14:57:28.385+0000] {processor.py:186} INFO - Started process (PID=63) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:57:28.386+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:57:28.388+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:57:28.388+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:57:28.393+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:57:28.392+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:57:28.393+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:57:28.410+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T14:57:58.679+0000] {processor.py:186} INFO - Started process (PID=64) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:57:58.679+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:57:58.682+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:57:58.681+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:57:58.687+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:57:58.686+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:57:58.687+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:57:58.700+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.027 seconds
[2025-03-13T14:58:29.006+0000] {processor.py:186} INFO - Started process (PID=65) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:58:29.009+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:58:29.013+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:58:29.012+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:58:29.026+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:58:29.024+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:58:29.027+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:58:29.070+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.080 seconds
[2025-03-13T14:58:59.229+0000] {processor.py:186} INFO - Started process (PID=66) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:58:59.230+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:58:59.232+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:58:59.232+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:58:59.241+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:58:59.240+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:58:59.241+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:58:59.262+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.041 seconds
[2025-03-13T14:59:29.455+0000] {processor.py:186} INFO - Started process (PID=67) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:59:29.456+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:59:29.459+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:59:29.459+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:59:29.466+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:59:29.465+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:59:29.466+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:59:29.488+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.041 seconds
[2025-03-13T14:59:59.704+0000] {processor.py:186} INFO - Started process (PID=68) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:59:59.705+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T14:59:59.709+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:59:59.708+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:59:59.718+0000] {logging_mixin.py:190} INFO - [2025-03-13T14:59:59.717+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T14:59:59.718+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T14:59:59.739+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.042 seconds
[2025-03-13T15:00:29.927+0000] {processor.py:186} INFO - Started process (PID=69) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:00:29.928+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:00:29.930+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:00:29.930+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:00:29.937+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:00:29.936+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:00:29.937+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:00:29.956+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.037 seconds
[2025-03-13T15:01:00.178+0000] {processor.py:186} INFO - Started process (PID=70) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:01:00.181+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:01:00.188+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:01:00.187+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:01:00.209+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:01:00.205+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:01:00.210+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:01:00.258+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.098 seconds
[2025-03-13T15:01:30.407+0000] {processor.py:186} INFO - Started process (PID=71) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:01:30.409+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:01:30.411+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:01:30.410+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:01:30.416+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:01:30.416+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:01:30.417+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:01:30.433+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.032 seconds
[2025-03-13T15:02:00.657+0000] {processor.py:186} INFO - Started process (PID=72) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:02:00.658+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:02:00.660+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:02:00.660+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:02:00.665+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:02:00.665+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:02:00.666+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:02:00.679+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T15:02:30.928+0000] {processor.py:186} INFO - Started process (PID=73) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:02:30.929+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:02:30.931+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:02:30.931+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:02:30.936+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:02:30.935+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:02:30.936+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:02:30.951+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T15:03:01.197+0000] {processor.py:186} INFO - Started process (PID=74) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:03:01.198+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:03:01.200+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:03:01.200+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:03:01.208+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:03:01.207+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:03:01.208+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:03:01.222+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T15:03:31.435+0000] {processor.py:186} INFO - Started process (PID=75) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:03:31.436+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:03:31.440+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:03:31.439+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:03:31.446+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:03:31.445+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:03:31.447+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:03:31.472+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.045 seconds
[2025-03-13T15:04:01.694+0000] {processor.py:186} INFO - Started process (PID=76) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:04:01.694+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:04:01.696+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:04:01.696+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:04:01.701+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:04:01.701+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:04:01.701+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:04:01.715+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.027 seconds
[2025-03-13T15:04:31.981+0000] {processor.py:186} INFO - Started process (PID=77) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:04:31.982+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:04:31.984+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:04:31.984+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:04:31.993+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:04:31.992+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:04:31.994+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:04:32.019+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.050 seconds
[2025-03-13T15:05:02.264+0000] {processor.py:186} INFO - Started process (PID=78) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:05:02.266+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:05:02.270+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:05:02.270+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:05:02.284+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:05:02.283+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:05:02.285+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:05:02.313+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.056 seconds
[2025-03-13T15:05:32.526+0000] {processor.py:186} INFO - Started process (PID=79) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:05:32.528+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:05:32.535+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:05:32.533+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:05:32.549+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:05:32.548+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:05:32.550+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:05:32.579+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.061 seconds
[2025-03-13T15:06:02.832+0000] {processor.py:186} INFO - Started process (PID=80) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:06:02.833+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:06:02.836+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:06:02.836+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:06:02.846+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:06:02.844+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:06:02.846+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:06:02.873+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.049 seconds
[2025-03-13T15:06:33.059+0000] {processor.py:186} INFO - Started process (PID=81) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:06:33.060+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:06:33.065+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:06:33.064+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:06:33.071+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:06:33.070+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:06:33.072+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:06:33.091+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.040 seconds
[2025-03-13T15:07:03.358+0000] {processor.py:186} INFO - Started process (PID=82) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:07:03.359+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:07:03.362+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:07:03.361+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:07:03.367+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:07:03.367+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:07:03.368+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:07:03.384+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.032 seconds
[2025-03-13T15:07:33.581+0000] {processor.py:186} INFO - Started process (PID=83) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:07:33.582+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:07:33.584+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:07:33.584+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:07:33.589+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:07:33.589+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:07:33.590+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:07:33.605+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:08:03.860+0000] {processor.py:186} INFO - Started process (PID=84) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:08:03.861+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:08:03.862+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:08:03.862+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:08:03.867+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:08:03.867+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:08:03.868+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:08:03.883+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T15:08:34.129+0000] {processor.py:186} INFO - Started process (PID=85) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:08:34.130+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:08:34.132+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:08:34.131+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:08:34.137+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:08:34.136+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:08:34.137+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:08:34.152+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T15:09:04.417+0000] {processor.py:186} INFO - Started process (PID=86) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:09:04.418+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:09:04.420+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:09:04.420+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:09:04.425+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:09:04.425+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:09:04.426+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:09:04.441+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:09:34.726+0000] {processor.py:186} INFO - Started process (PID=87) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:09:34.727+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:09:34.729+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:09:34.729+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:09:34.734+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:09:34.733+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:09:34.734+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:09:34.748+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.027 seconds
[2025-03-13T15:10:04.983+0000] {processor.py:186} INFO - Started process (PID=88) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:10:04.984+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:10:04.986+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:10:04.986+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:10:04.991+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:10:04.990+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:10:04.991+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:10:05.005+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.027 seconds
[2025-03-13T15:10:35.173+0000] {processor.py:186} INFO - Started process (PID=89) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:10:35.174+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:10:35.176+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:10:35.176+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:10:35.181+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:10:35.181+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:10:35.182+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:10:35.198+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T15:11:05.404+0000] {processor.py:186} INFO - Started process (PID=90) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:11:05.404+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:11:05.407+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:11:05.407+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:11:05.413+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:11:05.412+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:11:05.413+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:11:05.427+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T15:11:35.699+0000] {processor.py:186} INFO - Started process (PID=91) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:11:35.700+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:11:35.702+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:11:35.701+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:11:35.707+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:11:35.707+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:11:35.708+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:11:35.722+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T15:12:05.918+0000] {processor.py:186} INFO - Started process (PID=92) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:12:05.919+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:12:05.923+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:12:05.923+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:12:05.930+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:12:05.929+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:12:05.930+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:12:05.949+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.036 seconds
[2025-03-13T15:12:36.122+0000] {processor.py:186} INFO - Started process (PID=93) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:12:36.123+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:12:36.125+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:12:36.125+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:12:36.131+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:12:36.130+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:12:36.131+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:12:36.146+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:13:06.316+0000] {processor.py:186} INFO - Started process (PID=94) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:13:06.318+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:13:06.321+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:13:06.320+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:13:06.327+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:13:06.326+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:13:06.327+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:13:06.346+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.039 seconds
[2025-03-13T15:13:36.499+0000] {processor.py:186} INFO - Started process (PID=95) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:13:36.501+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:13:36.502+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:13:36.502+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:13:36.509+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:13:36.508+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:13:36.509+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:13:36.526+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.033 seconds
[2025-03-13T15:14:06.680+0000] {processor.py:186} INFO - Started process (PID=96) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:14:06.682+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:14:06.683+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:14:06.683+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:14:06.689+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:14:06.688+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:14:06.689+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:14:06.705+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:14:37.008+0000] {processor.py:186} INFO - Started process (PID=97) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:14:37.008+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:14:37.011+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:14:37.011+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:14:37.016+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:14:37.015+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:14:37.016+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:14:37.032+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T15:15:07.219+0000] {processor.py:186} INFO - Started process (PID=98) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:15:07.223+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:15:07.226+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:15:07.226+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:15:07.234+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:15:07.233+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:15:07.234+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:15:07.257+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.048 seconds
[2025-03-13T15:15:37.421+0000] {processor.py:186} INFO - Started process (PID=99) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:15:37.422+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:15:37.424+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:15:37.424+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:15:37.430+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:15:37.429+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:15:37.430+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:15:37.446+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.032 seconds
[2025-03-13T15:16:07.649+0000] {processor.py:186} INFO - Started process (PID=100) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:16:07.651+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:16:07.656+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:16:07.655+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:16:07.669+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:16:07.667+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:16:07.669+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:16:07.696+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.061 seconds
[2025-03-13T15:16:37.903+0000] {processor.py:186} INFO - Started process (PID=101) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:16:37.904+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:16:37.906+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:16:37.906+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:16:37.912+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:16:37.911+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:16:37.912+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:16:37.927+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T15:17:08.136+0000] {processor.py:186} INFO - Started process (PID=102) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:17:08.138+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:17:08.140+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:17:08.139+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:17:08.145+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:17:08.144+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:17:08.145+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:17:08.160+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T15:17:38.347+0000] {processor.py:186} INFO - Started process (PID=103) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:17:38.348+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:17:38.351+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:17:38.350+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:17:38.356+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:17:38.355+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:17:38.357+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:17:38.374+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.032 seconds
[2025-03-13T15:18:08.598+0000] {processor.py:186} INFO - Started process (PID=104) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:18:08.599+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:18:08.601+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:18:08.601+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:18:08.607+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:18:08.606+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:18:08.607+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:18:08.622+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:18:38.884+0000] {processor.py:186} INFO - Started process (PID=105) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:18:38.885+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:18:38.887+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:18:38.886+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:18:38.892+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:18:38.891+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:18:38.892+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:18:38.908+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:19:09.182+0000] {processor.py:186} INFO - Started process (PID=106) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:19:09.182+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:19:09.185+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:19:09.185+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:19:09.190+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:19:09.189+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:19:09.190+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:19:09.207+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:19:39.484+0000] {processor.py:186} INFO - Started process (PID=107) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:19:39.485+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:19:39.487+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:19:39.486+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:19:39.492+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:19:39.491+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:19:39.492+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:19:39.507+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T15:20:09.762+0000] {processor.py:186} INFO - Started process (PID=108) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:20:09.763+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:20:09.766+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:20:09.765+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:20:09.772+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:20:09.771+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:20:09.772+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:20:09.790+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.034 seconds
[2025-03-13T15:20:40.090+0000] {processor.py:186} INFO - Started process (PID=109) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:20:40.090+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:20:40.092+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:20:40.092+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:20:40.097+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:20:40.097+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:20:40.098+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:20:40.114+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:21:10.383+0000] {processor.py:186} INFO - Started process (PID=110) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:21:10.384+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:21:10.386+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:21:10.385+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:21:10.391+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:21:10.390+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:21:10.391+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:21:10.405+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.027 seconds
[2025-03-13T15:21:40.693+0000] {processor.py:186} INFO - Started process (PID=111) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:21:40.694+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:21:40.696+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:21:40.696+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:21:40.701+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:21:40.701+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:21:40.701+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:21:40.717+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:22:10.945+0000] {processor.py:186} INFO - Started process (PID=112) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:22:10.945+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:22:10.947+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:22:10.947+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:22:10.953+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:22:10.951+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:22:10.953+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:22:10.970+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:22:41.237+0000] {processor.py:186} INFO - Started process (PID=113) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:22:41.238+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:22:41.240+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:22:41.240+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:22:41.246+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:22:41.245+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:22:41.246+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:22:41.262+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:23:11.530+0000] {processor.py:186} INFO - Started process (PID=114) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:23:11.531+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:23:11.534+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:23:11.533+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:23:11.539+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:23:11.538+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:23:11.539+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:23:11.554+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T15:23:41.830+0000] {processor.py:186} INFO - Started process (PID=115) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:23:41.830+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:23:41.832+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:23:41.832+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:23:41.837+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:23:41.837+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:23:41.838+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:23:41.851+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T15:24:12.147+0000] {processor.py:186} INFO - Started process (PID=116) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:24:12.147+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:24:12.150+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:24:12.149+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:24:12.155+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:24:12.154+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:24:12.155+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:24:12.171+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:24:42.484+0000] {processor.py:186} INFO - Started process (PID=117) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:24:42.485+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:24:42.487+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:24:42.487+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:24:42.492+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:24:42.492+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:24:42.493+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:24:42.509+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:25:12.797+0000] {processor.py:186} INFO - Started process (PID=118) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:25:12.798+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:25:12.800+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:25:12.800+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:25:12.805+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:25:12.805+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:25:12.806+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:25:12.822+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:25:43.078+0000] {processor.py:186} INFO - Started process (PID=119) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:25:43.078+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:25:43.081+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:25:43.081+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:25:43.086+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:25:43.085+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:25:43.086+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:25:43.101+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T15:26:13.370+0000] {processor.py:186} INFO - Started process (PID=120) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:26:13.371+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:26:13.373+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:26:13.373+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:26:13.378+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:26:13.378+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:26:13.379+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:26:13.393+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T15:26:43.682+0000] {processor.py:186} INFO - Started process (PID=121) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:26:43.683+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:26:43.685+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:26:43.684+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:26:43.690+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:26:43.689+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:26:43.690+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:26:43.706+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:27:13.996+0000] {processor.py:186} INFO - Started process (PID=122) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:27:13.997+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:27:14.000+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:27:13.999+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:27:14.005+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:27:14.004+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:27:14.005+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:27:14.022+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T15:27:44.310+0000] {processor.py:186} INFO - Started process (PID=123) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:27:44.310+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:27:44.313+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:27:44.312+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:27:44.317+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:27:44.317+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:27:44.318+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:27:44.333+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:28:14.600+0000] {processor.py:186} INFO - Started process (PID=124) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:28:14.601+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:28:14.603+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:28:14.603+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:28:14.609+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:28:14.608+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:28:14.609+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:28:14.624+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T15:28:44.903+0000] {processor.py:186} INFO - Started process (PID=125) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:28:44.904+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:28:44.907+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:28:44.906+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:28:44.912+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:28:44.911+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:28:44.913+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:28:44.926+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T15:29:15.198+0000] {processor.py:186} INFO - Started process (PID=126) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:29:15.199+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:29:15.201+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:29:15.200+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:29:15.207+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:29:15.206+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:29:15.207+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:29:15.220+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T15:29:45.513+0000] {processor.py:186} INFO - Started process (PID=127) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:29:45.513+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:29:45.516+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:29:45.515+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:29:45.521+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:29:45.520+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:29:45.521+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:29:45.537+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T15:30:15.806+0000] {processor.py:186} INFO - Started process (PID=128) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:30:15.807+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:30:15.809+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:30:15.809+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:30:15.815+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:30:15.814+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:30:15.815+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:30:15.829+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T15:30:46.094+0000] {processor.py:186} INFO - Started process (PID=129) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:30:46.095+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:30:46.097+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:30:46.097+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:30:46.102+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:30:46.101+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:30:46.102+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:30:46.118+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T15:31:16.393+0000] {processor.py:186} INFO - Started process (PID=130) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:31:16.394+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:31:16.396+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:31:16.396+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:31:16.401+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:31:16.401+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:31:16.402+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:31:16.418+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.030 seconds
[2025-03-13T15:31:46.701+0000] {processor.py:186} INFO - Started process (PID=131) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:31:46.701+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:31:46.703+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:31:46.703+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:31:46.709+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:31:46.708+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:31:46.709+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:31:46.723+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.028 seconds
[2025-03-13T15:32:16.995+0000] {processor.py:186} INFO - Started process (PID=132) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:32:16.996+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:32:16.998+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:32:16.998+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:32:17.003+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:32:17.003+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:32:17.004+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:32:17.021+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
[2025-03-13T15:32:47.295+0000] {processor.py:186} INFO - Started process (PID=133) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:32:47.296+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:32:47.298+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:32:47.298+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:32:47.303+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:32:47.303+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:32:47.304+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:32:47.319+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T15:33:17.592+0000] {processor.py:186} INFO - Started process (PID=134) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:33:17.593+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:33:17.595+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:33:17.595+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:33:17.601+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:33:17.600+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:33:17.601+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:33:17.614+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.027 seconds
[2025-03-13T15:33:47.898+0000] {processor.py:186} INFO - Started process (PID=135) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:33:47.899+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:33:47.901+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:33:47.901+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:33:47.906+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:33:47.906+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:33:47.906+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:33:47.920+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.027 seconds
[2025-03-13T15:34:18.197+0000] {processor.py:186} INFO - Started process (PID=136) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:34:18.198+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:34:18.200+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:34:18.200+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:34:18.205+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:34:18.205+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:34:18.206+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:34:18.221+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.029 seconds
[2025-03-13T15:34:48.475+0000] {processor.py:186} INFO - Started process (PID=137) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:34:48.476+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:34:48.478+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:34:48.478+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:34:48.483+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:34:48.482+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:34:48.483+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:34:48.497+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.027 seconds
[2025-03-13T15:35:18.728+0000] {processor.py:186} INFO - Started process (PID=138) to work on /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:35:18.729+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/reddit_dag.py for tasks to queue
[2025-03-13T15:35:18.731+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:35:18.731+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:35:18.736+0000] {logging_mixin.py:190} INFO - [2025-03-13T15:35:18.736+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/reddit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/reddit_dag.py", line 28, in <module>
    python_callable=reddit_pipeline,
NameError: name 'reddit_pipeline' is not defined
[2025-03-13T15:35:18.737+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/reddit_dag.py
[2025-03-13T15:35:18.753+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/reddit_dag.py took 0.031 seconds
